chore(electron): configure auto-updater to not use "v"-prefix

SKoschnicke opened the pull request:
The default setting for the electron-autoupdater when using GitHub is to
check for tags with a prefixed "v" and the version. This can be switched
off by calling setFeedURL[1] with GitHub options hash[2]. Unfortunately, this
leads to repeating the repository name and owner there, which is
otherwise detected automatically from the package.json.

[1] https://www.electron.build/auto-update#module_electron-updater.AppUpdater+setFeedURL
[2] https://www.electron.build/configuration/publish#githuboptions

SKoschnicke committed:
chore(electron): configure auto-updater to not use "v"-prefix

The default setting for the electron-autoupdater when using GitHub is to
check for tags with a prefixed "v" and the version. This can be switched
off by calling setFeedURL[1] with GitHub options hash[2]. Unfortunately, this
leads to repeating the repository name and owner there, which is
otherwise detected automatically from the package.json.

[1] https://www.electron.build/auto-update#module_electron-updater.AppUpdater+setFeedURL
[2] https://www.electron.build/configuration/publish#githuboptions

Yasamato reviewed (approved):
Es gibt eine Sache die ich noch mal ansprechen möchte: User mit aktueller Updatefunktion mit v. werden keinen Autoupdate tätigen können, da der nächste Release mit ohne v. getagt sein wird korrekt? @SKoschnicke @Xerus2000 
Es ist also ein manuelles Update via github oder der Webseite nötig. Des weiteren werden bei allen bisherigen Usern dann bei jedem Start eine Fehlermeldung angezeigt, was ich für verkraftbar halte, da somit Teilnehmer auf das Update aufmerksam werden.

Yasamato committed:
chore(tagging): implemented yarn version

Run once `yarn set-config`.
For release-tagging, run `yarn version --<major|minor|patch>` .
For more details see: https://yarnpkg.com/lang/en/docs/cli/version/#toc-commands

Agent77326 commented:
@SKoschnicke Ich hab das Release-Verfahren für die GUI angepasst, siehe Notion unter "Abläufe>Creating a Release".

Xerus2000 commented:
@Agent77326 verlinke doch bei sowas auch direkt die Notion-Seite ;)
https://www.notion.so/softwarechallenge/Creating-a-Release-1732217fb0234469b3d5653436f357db
